custom/icds_reports/reports/awcs_covered.py

```diff
@@ -111,7 +111,7 @@ def get_data_for(filters):
             "Number of {:s} launched: {:s}".format(
                 awcs_launched_help_text(),
                 indian_formatted_number(total_awcs),
-                prop.title(),
+                'Sectors' if prop.title() == 'Supervisors' else prop.title(),
```

Comment:
Is there a need to keep the `Supervisors` in the code in the function. Could we just use sector starting from the annotate in the query Instead of putting this special condition here?

Reply:
Since it was just a Nomenclature change to be displayed in UI, I thought it would be good to not break the consistent naming convention. 
(Partly, I'm fairly new working with django. So, I made the change only where it was absolutely necessary)

Reply:
Hey, I think `supervisors` is used in this file only and in case_Data_reach one. and if we want sectors instead of that I think its good to change it to that instead of having both of then. Special because, having supervisors in the query, it needs you to specially check for it and change. 
 
do you know about the priority of this task?If not high priority, It would be good to change it in the query level only.  Also, good to add the test case for this so that we are sure that this does what you want.

Reply:
Hey, I think supervisors is used in this file only and in case_Data_reach one. and if we want sectors instead of that I think its good to change it to that instead of having both of then. Special because, having supervisors in the query, it needs you to specially check for it and change.

do you know about the priority of this task?If not high priority, It would be good to change it in the query level only. Also, good to add the test case for this so that we are sure that this does what you want.

Reply:
Hey, I made change in cas_data_reach file also. (In that file, label was being sent as Sector to frontend, if location is "supervisor". I have refactored that to "Sector" now).

I'm not comfortable enough with django to make change at query level. So, I made change starting from annotate.